src/nrfx/drivers/nrfx_common.h

```diff
@@ -50,6 +50,16 @@
 extern "C" {
 #endif
 
+/*
+ * Suppress use of anomaly workarounds in nrfx drivers that would directly
+ * access hardware registers.
+ */
+#define USE_WORKAROUND_FOR_ANOMALY_132 0
+
+#ifndef __STATIC_INLINE
```

Comment:
Just stepping a bit in your toes: Should the real nrfx be stop using this macro and instead consistantly use NRFX_STATIC_INLINE?

Anyhow, this definition can be here, but maybe it would be good to add a comment saying why we need it. Something like "In the real nrfx hal, this macro is inherited from CMSIS."

Reply:
> Just stepping a bit in your toes: Should the real nrfx be stop using this macro and instead consistantly use NRFX_STATIC_INLINE?

No, we use those macros separately because `NRFX_STATIC_INLINE` is sometimes (for example in unit tests) defined as empty.

> Anyhow, this definition can be here, but maybe it would be good to add a comment saying why we need it. Something like "In the real nrfx hal, this macro is inherited from CMSIS."

Good point. I'll add such comment.

Reply:
Added.